geoportal/c2cgeoportal_geoportal/scaffolds/update/geoportal/CONST_config-schema.yaml

```diff
@@ -178,6 +178,16 @@ mapping:
                 type: str
               do_redirect:
                 type: bool
+      authentication:
+        type: map
+        required: false
+        mapping:
+          two_factor:
+            type: bool
+            required: false
+          max_consecutive_failed:
+            type: int
+            required: false
```

Comment:
Not really related with this PR but how about "two_factor_issuer_name" ? seems that two_factor_issuer_name is also missing in doc.

Reply:
+1 :-)